doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -2992,9 +2994,60 @@ Deletes an ``AuthenticatedUser`` whose ``id``  is passed. ::
+    
+Disables an ``AuthenticatedUser`` whose ``id``  is passed. ::
+
+    PUT http://$SERVER/api/admin/authenticatedUsers/disable/id/$id
     
+Note: Since the primary purpose of Dataverse is to serve as an archive and to track data accesses and the modifications of data and metadata, a simple mechanism to delete users that have performed certain actions in the system is not provided by design. 
+
+Disabling a user with this endpoint will:
+
+- Disable the user's ability to log in to the Dataverse installation
+- Remove the user's access from all datasets and files
+- Cancel any pending file access requests generated by the user
+- Remove the user from all groups
+- No longer have notifications generated or sent by the Dataverse installation
+
+Disabling a user with this endpoint will keep:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author.
+- Any Dataverse Collections where the user is the sole administrator
+- The user's access to datafiles in the Dataverse installation, including guestbook records and acceptance of terms of use.
+- The user's account information (specifically name, email, affiliation, and position)
+
+If it is determined that disabling a user is not sufficient, it is possible anonymize a user through use of the :ref:`merge-accounts-label` feature. See :ref:`Anonymize a User <disable-a-user>` for more information about this process. 
+
+.. _anonymize-a-user:
+
+Anonymize a User
+~~~~~~~~~~~~~~~~
+
+In some cases, due to privacy laws or other organizational policies, it may be required to anonymize a user. This process should only be used in cases where a Dataverse installation has the need to completely remove the traces of a user from the Dataverse installation. This process is only provided because it is not feasible for the Dataverse Software to provide deletion and deactivation options that will satisfy all use cases and all privacy laws, so the most complete deletion option is provided here. 
+
+Note: Before going through these steps, please note that you will be permanently losing:
+
+- The user's contributions to datasets, including dataset creation, file uploads, publishing, submitting for review, and returning to author. This will all be marked as "Anonymous User" in your Dataverse installation, and it will not be possible to track who made changes as part of the archival record.
+- Records of the user's access to datafiles in the Dataverse installation, including guestbook records and acceptance of terms of use. This could potentially introduce issues where it is no longer possible for data curators and owners to follow up with those who have accessed data on the application, which may be a legal requirement or data sharing agreement. It will be up to the Dataverse installation administrators to determine which datasets the user has accessed to determine if anonymization is feasible in these situations.    
```

Comment:
we would still have their emails in the guestbooks in this case.

Reply:
updated in https://github.com/IQSS/dataverse/pull/7585/commits/2af33fd9662328d86518c6f8c985c187398d1a0b, by pulling out into a bullet of stuff that's kept during this process. I continue my lean towards removing the anonymize section of the docs.